src/MoBi.Presentation/Presenter/EditMoleculeBuilderPresenter.cs

```diff
@@ -105,7 +105,7 @@ public void Edit(MoleculeBuilder moleculeBuilder, IReadOnlyList<IObjectBase> exi
 
       public void Edit(MoleculeBuilder moleculeBuilder)
       {
-         Edit(moleculeBuilder, moleculeBuilder.ParentContainer.Children);
+         Edit(moleculeBuilder, moleculeBuilder.ParentContainer?.Children);
```

Comment:
It was just this ParentContainer can be null. In develop branch, a null value is acceptable in the second parameter of Edit

Reply:
what's the parent container of a molecule builder? I don't get it